lib/sgf/gtp_writer.rb

```diff
@@ -0,0 +1,31 @@
+class SGF::GtpWriter < SGF::Writer
+
+  private
+
+  def gtp_move(node)
+    pps = node.properties
+    return "boardsize #{pps['SZ']}\nclear_board" if pps['SZ']
+    return "" if pps.size > 1 || (pps.keys != ["B"] && pps.keys != ["W"])
```

Comment:
This reads to me as "If there is more than one property, return an empty string. Also, if there is more than one property and it's not Black _AND_ it's not White, return an empty string". That means you couldn't send nodes that have comments on them. Is that intended? Or would we want to strip comments instead?

Reply:
i only need the boardsize and the moves. GTP is the Go Text Protocol, a way to communicate a game move by move to a software. most AI use GTP to talk to servers or desktop clients

with this gem and my changes, I can loaded up any SGF and send it to any AI I want